protocols/manage/count.go

```diff
@@ -106,9 +106,11 @@ func (p *ProtocolCount) FuncPC() {
 		p.SendTo(p.Parent(), &NodeIsUp{})
 	}
 	if !p.IsLeaf() {
-		for _, c := range p.Children() {
-			dbg.Lvl3(p.Info(), "sending to", c.Entity.Addresses, c.Id, p.timeout)
-			p.SendTo(c, &PrepareCount{Timeout: p.timeout})
+		for _, child := range p.Children() {
+			go func(c *sda.TreeNode) {
```

Comment:
just out of curiosity (already asked that question I think but can't find it anywhere):
Why use a go routine ? I'm not against go routine at all but it should be avoided when it can be avoided.